apps/odyssey-stats/src/widget/hightlights.scss

```diff
@@ -19,7 +19,8 @@ $offset-padding-hovering: 8px;
 		display: none;
 		margin-top: 16px;
 
-		@media only screen and (max-width: $break-dashboard-mobile) {
+		/* stylelint-disable-next-line scss/at-rule-no-unknown */
+		@container (max-width: #{$break-dashboard-mobile}) {
```

Comment:
We may want to consider using `$break-small` (600px) from `@wordpress/base-styles/breakpoints` instead of defining our own breakpoint. Doing so will allow us to avoid adding stylelint exceptions.

Reply:
The linting error is from `@container` which is unknown to SCSS transpiler. I did change to use `@wordpress/base-styles/breakpoints`, but the error deprecating is still needed to pass linting.